lib/mdx.ts

```diff
@@ -15,8 +15,8 @@ export const mdx = (tree: any, { hast = false } = {}) => {
     .use(divTransformer)
     .use(readmeToMdx)
     .use(tablesToJsx)
-    .use(remarkStringify)
-    .use(compilers);
+    .use(compilers)
+    .use(remarkStringify);
```

Comment:
We're not doing it yet, but this changing the order allows us to override the defaults.

Reply:
~Ahh, so is this how you were able to [delete the image compiler](https://github.com/readmeio/markdown/pull/951/files?show-deleted-files=true&show-viewed-files=true&file-filters%5B%5D=#diff-5e6825f007ffc0385b84a1770dd7496b4e7b9dea0fdd206860715060f1cc8ed8) wholesale?~ Or wait, it looks like we just [using the custom component *instead* of](https://github.com/readmeio/markdown/pull/951/files?show-deleted-files=true&show-viewed-files=true&file-filters%5B%5D=#diff-c626b24d42bb64ab7faddfb1f3a3e11f55b8dec3e952a1f13540ba7dd9621f13R48) that old custom compiler? This definitely feels more MDX-y, I think!

Reply:
Yea, this PR detects earlier (with a transformer) if it needs MDX syntax. That way we're relying on the default compilers for both MDX images and markdown images.